packages/core/src/Site/index.js

```diff
@@ -1432,6 +1432,7 @@ class Site {
           title: page.title,
           headings: page.headings,
           headingKeywords: page.keywords,
+          frontmatter: page.frontMatter,
```

Comment:
I'm thinking we should add in just the keywords for now, I don't see the other properties being of use. What do you think?

Reply:
I agree! I will make the necessary changes in the next commit!